Theme.html

```diff
@@ -884,7 +885,11 @@
 	</head>
 	<body>
 		<script>
-			document.body.innerHTML = `<div>${translate("", data)}</div>`;
+			document.body.innerHTML = `<div>${
+			    render(data, {
+					autoOpenDepth: {text:AutoExpand Depth},
```

Comment:
this whole line shouldn't be rendered if the option is left blank, i.e. should be wrapped in `{block:IfAutoExpandDepth}` `{/block:IfAutoExpandDepth}`

bad inputs breaking things is fine but blank inputs breaking things makes me sad